build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -22,6 +22,145 @@ data:
     & antrea-agent --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
   Run-AntreaOVS-Containerd.ps1: |
     $ErrorActionPreference = "Stop"
+    $OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-3.0.5-antrea.0-win64.zip"
+    $OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+    $WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+    $OVSDownloadDir = $WorkDir
+    $OVSInstallDir = "C:\openvswitch"
+    $OVSZip = "$OVSDownloadDir\ovs-win64.zip"
+    $ImportCertificate = $true
+    $CheckFileHash = $true
+    $PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+    $InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+    function Log($Info) {
+        $time = $(get-date -Format g)
+        "$time $Info `n`r" | Tee-Object $InstallLog -Append | Write-Host
+    }
+
+    function CreatePath($Path){
+        if ($(Test-Path $Path)) {
+            mv $Path $($Path + "_bak")
+        }
+        mkdir -p $Path | Out-Null
+    }
+
+    function SetEnvVar($key, $value) {
+        [Environment]::SetEnvironmentVariable($key, $value, [EnvironmentVariableTarget]::Machine)
+    }
+
+    function WaitExpandFiles($Src, $Dest) {
+        Log "Extract $Src to $Dest"
+        Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+    }
+
+    function ServiceExists($ServiceName) {
+        If (Get-Service $ServiceName -ErrorAction SilentlyContinue) {
+            return $true
+        }
+        return $false
+    }
+
+    function CheckIfOVSInstalled() {
+        if (Test-Path -Path $OVSInstallDir) {
+            Log "$OVSInstallDir already exists, exit OVS installation."
+            exit 1
+        }
+        If (ServiceExists("ovs-vswitchd")) {
+            Log "Found existing OVS service, exit OVS installation."
+            exit 0
+        }
+    }
```

Comment:
checkifovsinstalled is not required ,
no need for service existence on host for ovs-vswitchd

Reply:
removed service exists function, but we need to check whether openvswitch directory exists or not in case of container restart